docs/developer/extending/webhooks/synchronous-events/tax.mdx

```diff
@@ -443,3 +447,11 @@ The payload sent to the app would look like this:
    1. `tax_rate` - Tax rate value provided as percentage. Example: provide `23` to represent the `23%` tax rate.
    2. `total_gross_amount` - Gross price of the line.
    3. `total_net_amount` - Net price of the line.
+
+
+## Tax data validation
+
+The app response is considered valid if:
+1. There are no negative values
+2. Request's number of lines matches number of lines from response
+3. The prices and rates do not exceed database limits
```

Comment:
Would be better to explicitly say: that prices should have maximal X numbers after coma etc. Right now it doesn't say anything as you need to be familiar with Saleor core code, to find the limits. 
Btw. Do we have such limitation? I though that we will get the price amount and apply rounding on it to reduce the decimal numbers 🤔